### PR TITLE
Version 0.1 of Auth software.

### PR DESCRIPTION
**Pull request by rmbarron**
This is to add Version 0.1 of the software portion.

### CODE REVIEW

File: software/main.py [hunk 1]
```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/python
```
Comment (thatch) [inline software/main.py:1]: The license block is missing.

File: software/findKeyboards.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/python
```
Comment (thatch) [inline software/findKeyboards.py:1]: The license block is missing

File: software/findKeyboards.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+import evdev
+
+def main():
+	print("Input devices connected to machine:")
+	devices = [evdev.InputDevice(x) for x in evdev.list_devices()]
+	for i in devices:
+		print devices[devices.index(i)]
+
+if __name__ == "__main__":
+	main()
```
Comment (thatch) [inline software/findKeyboards.py]: Add trailing newline, fix indentation to be reasonable (like, 4 spaces per PEP8 or 2 spaces per gpystyle)

File: software/main.py [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+	self.press_event.clear()
+	self.pressed = False
+
+    def _callback(self, channel):
+	self.press_event.set()
+        self.pressed = True
+
+# looks at connected devices, returns device matching name given
+def get_scanner_device():
+	deviceName = '<NameOfUSBBadgeScanner>'
```
Comment (thatch) [inline software/main.py]: here and below, fix indentation to be consistent

**Comment by rmbarron** [comment]
Appropriate blocks were added and formatting was changed to 2-space indents

**Pull request merged**